tests/e2e/e2e.go

```diff
@@ -517,6 +428,15 @@ func (test *TestCtx) AlreadyRunningMicroservice(msName string) *Microservice {
 	return ms
 }
 
+func (test *TestCtx) AlreadyRunningAgent(agentName string) *Agent {
```

Comment:
This should be named `GetRunningAgent` to be more descriptive about what it does. Function name `AlreadyRunningAgent` seems more like it would return `bool` value if the agent is running already.